web/src/recent_view_ui.js

```diff
@@ -1492,6 +1492,9 @@ export function initialize({
         $("#recent_view_search").val("");
         update_filters_view();
     });
+    $("body").on("click", "#compose_close", () => {
+        setTimeout(revive_current_focus, 0);
+    });
```

Comment:
What if user uses `escape` to close compose?

Reply:
it seems not working when users uses escape thank you for informing